text/0001-rfc-process.md

```diff
@@ -0,0 +1,428 @@
+  - The FCP lasts ten calendar days, so that it is open for at least 5 business
+    days. It is also advertised widely,
+    e.g. on the LF mailing list. This way all
+    stakeholders have a chance to lodge any final objections before a decision
+    is reached.
+  - In most cases, the FCP period is quiet, and the RFC is either merged or
+    closed. However, sometimes substantial new arguments or ideas are raised,
+    the FCP is canceled, and the RFC goes back into development mode.
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+Once an RFC becomes "active" then authors may implement it and submit the
+feature as a pull request to the relevant repo. Being "active" is not a rubber
+stamp, and in particular still does not mean the feature will ultimately be
+merged; it does mean that in principle all the major stakeholders have agreed
+to the feature and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted and is "active"
+implies nothing about what priority is assigned to its implementation, nor does
+it imply anything about whether a developer has been assigned the task of
+implementing the feature. While it is not *necessary* that the author of the
+RFC also write the implementation, it is by far the most effective way to see
+an RFC through to completion: authors should not expect that other project
+developers will take on responsibility for implementing their accepted feature.
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We
+strive to write each RFC in a manner that it will reflect the final design of
+the feature; but the nature of the process means that we cannot expect every
+merged RFC to actually reflect what the end result will be at the time of the
+next major release.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes
+should be new RFCs, with a note added to the original RFC. Exactly what counts
+as a "very minor change" is up to the team to decide.
+
+## Reviewing RFCs
+[Reviewing RFCs]: #reviewing-rfcs
+
+While the RFC pull request is up, the core developers may schedule meetings with
+the author and/or relevant stakeholders to discuss the issues in greater detail,
+and in some cases the topic may be discussed at a regular LF meeting. In either
+case a summary from the meeting will be posted back to the RFC pull request.
+
+Final decisions about RFCs are made after the benefits and drawbacks are well
+understood. These decisions can be made at any time. When a decision is made,
+the RFC pull request will either be merged or closed. In either case, if the
+reasoning is not clear from the discussion in the thread, the a comment
+describing the rationale for the decision will be added.
+
+
+## Implementing an RFC
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital features that need to be implemented right
```

Comment:
I would like to raise the point about what is officially supported. 

-> What are the sections of code that always should have feature parity?

Reply:
Okay, let us say we approve a new RFC that adds some runtime/language features. The question is, what are significant changes and what are not? I would argue that we should define a language core and that any RFC that changes the language core should be merged in one go. I think we'd be shooting ourselves in the foot if a Lingua Franca release has only a half-functioning feature (e.g. C target has support, but CPP doesn't).

Reply:
I think that this might be too tall of an order. It rules out features that cannot be supported by all targets and it could cause features to be delayed indefinitely if maintainers of some targets prioritize other things...

Reply:
Its also an orthogonal question in my opinion. The proposed RFC process explicitly focuses on features, not implementations. Accepted (active) RFCs are a commitment of the community to a specific feature. It does not mean that there is or will be a complete implementation of that feature in the near term. It could also be that an active RFC is never implemented because it simply isn't a priority.

The point you raise is important, too. How do we manage the heterogeneity of features between the LF targets? How do we ensure some level of feature parity? These are beyond the scope of the RFC process as proposed here. I also don't see a clear way for including these questions in our process. Do you have a concrete proposal? Otherwise, I would say this is a great topic for a future RFC.